src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeImageSection.ts

```diff
@@ -0,0 +1,71 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import { h } from 'react-hyperscript-helpers';
+import { GcpComputeImageSelect } from 'src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeImageSelect';
+import { ComputeImage, ComputeImageStore, useComputeImages } from 'src/analysis/useComputeImages';
+import { getImageUrlFromRuntime } from 'src/analysis/utils/runtime-utils';
+import { RuntimeToolLabel, runtimeToolLabels, runtimeTools } from 'src/analysis/utils/tool-utils';
+import { spinnerOverlay } from 'src/components/common';
+import { GetRuntimeItem } from 'src/libs/ajax/leonardo/models/runtime-models';
+
+export interface GcpComputeImageSectionProps {
+  readonly onSelect: (image: ComputeImage | undefined, isCustomImage: boolean) => void;
+  readonly tool: RuntimeToolLabel;
+  readonly currentRuntime?: Pick<GetRuntimeItem, 'runtimeImages'>;
+}
+
+const customImageOptionUrl = 'CUSTOM_IMAGE_OPTION';
+
+export const GcpComputeImageSection: React.FC<GcpComputeImageSectionProps> = (props: GcpComputeImageSectionProps) => {
```

Comment:
```suggestion
export const GcpComputeImageSection: React.FC<GcpComputeImageSectionProps> = (props) => {
```

or

```suggestion
export const GcpComputeImageSection = (props: GcpComputeImageSectionProps) => {
```

Nit: the props type doesn't need to be specified twice.